evap/staff/templates/staff_semester_view.html

```diff
@@ -444,16 +444,16 @@ <h3>
             $(this).find('.btn').toggleClass('btn-secondary');
         });
 
-        if (localStorage['collapse_overview'] === 'false') {
-            $('#overviewCard').collapse('hide');
+        if (localStorage['show_overview'] === 'true') {
+            $('#overviewCard').collapse('show');
         }
         else {
-            $('#overviewCard').collapse('show');
+            $('#overviewCard').collapse('hide');
         }
 
         function saveCollapseState() {
-            var state = window.localStorage['collapse_overview'] === 'true';
-            localStorage['collapse_overview'] = (!state).toString();
+            var state = window.localStorage['show_overview'] === 'true';
```

Comment:
Is there a reason to use `window.localStorage` here when we just use `localStorage` everywhere else? Confused me quite a bit.

Reply:
i can't see any reason, i'll remove it.